awx/main/models/organization.py

```diff
@@ -123,6 +126,10 @@ def get_absolute_url(self, request=None):
     def _get_related_jobs(self):
         return UnifiedJob.objects.non_polymorphic().filter(organization=self)
 
+    def validate_role_assignment(self, actor, role_definition):
+        if role_definition.name in ['Organization Admin', 'Organization Member']:
+            raise DRFValidationError({'detail': _(f"Assignment must use the Controller {role_definition.name} role.")})
```

Comment:
Remove this validation. It'll be handled in DAB and has to be configurable.

Reply:
With the new DAB version, you should be able to add the setting that will replace this:

```python
ALLOW_LOCAL_ASSIGNING_JWT_ROLES = False
```